src/index.json

```diff
@@ -941,7 +941,7 @@
                     },
                     "extras": [],
                     "generator": "bdist_wheel (0.30.0)",
-                    "license": "Proprietary https://aka.ms/azureml-preview-sdk-license ",
+                    "license": "Proprietary https://aka.ms/azureml-preview-sdk-license",
```

Comment:
This is autogenerated. So I guess this will be reverted once there's a new version of azure-cli-ml.